src/modeling/train_cnn.py

```diff
@@ -58,25 +101,18 @@ def run_cnn(split_directory, batchsize=256, epochs=50, color_mode='rgb'):
                                                              classes={'no_fishing': 0, 'fishing': 1}, 
                                                              shuffle=False)
 
-    # Creating basic tf.keras sequential model
-    model = Sequential([
-        Conv2D(8, 3, padding='same', activation='relu', input_shape=(IMG_HEIGHT, IMG_WIDTH ,IMG_DEPTH)),
-        Conv2D(8, 3, padding='same', activation='relu'),
-        MaxPooling2D(),
-        Conv2D(16, 3, padding='same', activation='relu'),
-        Conv2D(16, 3, padding='same', activation='relu'),
-        MaxPooling2D(),
-        Conv2D(32, 3, padding='same', activation='relu'),
-        Conv2D(32, 3, padding='same', activation='relu'),
-        MaxPooling2D(),
-        Flatten(),
-        Dense(256, activation='relu'),
-        Dense(256, activation='relu'),
-        Dense(1, activation='sigmoid')
-    ])
+    # Creating tf.keras functional model
+    image_input = Input(shape=(IMG_HEIGHT, IMG_WIDTH , IMG_DEPTH))
+    layer = Conv_block(image_input, start_filters, 2, batchnorm=bnorm)
+    for i in range(depth):
+        layer = Conv_block(layer, start_filters*(2*(i+1)), 2, batchnorm=bnorm)
+    layer = Flatten()(layer)
+    layer = Dense_block(layer, dense_neurons,dense_count)
+    layer = Dense(1, activation='sigmoid')(layer)
+    model = Model(inputs=image_input, outputs=layer)
 
     # Early stopping
-    earlystop = EarlyStopping(monitor='val_loss', restore_best_weights=True, patience = 4)
+    earlystop = EarlyStopping(monitor='val_loss', restore_best_weights=True, patience = 8)
```

Comment:
Agreed that 8 is a fine default. Perhaps this param should be passed in when run_model is called, though?